containers/runner/launch

```diff
@@ -60,12 +61,14 @@ Options:
  --recommended-jobs                   Print the number of jobs (--jobs option) used
                                       by default and exit. It is calulated from the number
                                       of CPUs and the size of RAM.
+ --kstest-group NAME                  Name of the tests being run, used to group results in json
+                                      logs.
```

Comment:
I don't want be to too nitpicky and I'd hate to block your work but I'd still use rather some generic option name. To me it seems that - as you say in the description of the option - it is just a name. From the launcher perspective it is not used to group results, it is used to name/tag (all) the results of this launch run. I'd name it --run-name. But I am not going to block the PR and on it as I said.

Reply:
Hmm, but then why not name it scenario? :) I don't really like `--run-name` since that implies that it is tied to the specific run, but it isn't since multiple runs will have the same scenario name. Naming is hard, so I'm certainly open to more suggestions. It just can't be named 'name' since that is already used for the name of the specific test for each entry.

Reply:
I agree that if we go concrete here, then the --scenario is better :)